packages/ui/src/machines/authMachine.ts

```diff
@@ -129,8 +132,9 @@ export const authMachine = createMachine<AuthContext, AuthEvent>(
         actorRef: (context, event) => {
           const actor = signUpActor.withContext({
             authAttributes: event.data?.authAttributes ?? {},
+            country_code: DEFAULT_COUNTRY_CODE,
             intent: event.data?.intent,
-            formValues: { country_code: '+1' },
+            formValues: {},
             validationError: {},
             login_mechanisms: context.config?.login_mechanisms,
```

Comment:
nit - would be nice to pull out the common default context values here into a separate object for reuse across the machine. Looks like we use most of these default values in a couple places.

Reply:
@eddiekeller By "default context values" do you mean the ones we hard code in a couple of the actor spawners? So what you suggest would look something like:
```javascript
const DEFAULT_CONTEXT_VALUES = {
  country_code: '+1',
  formValues: {},
  validationError: {},
};

...

      spawnSignInActor: assign({
        actorRef: (_, event) => {
          const actor = signInActor.withContext({
            authAttributes: event.data?.authAttributes,
            user: event.data?.user,
            intent: event.data?.intent,
            ...DEFAULT_CONTEXT_VALUES,
          });
          return spawn(actor, { name: 'signInActor' });
        },
      }),
```

Reply:
Something like that. You don't have to do it for this PR. It was just a thought I had while reading through the code.